### PR TITLE
Error 404 page

### PR DESCRIPTION
**Pull request by rdev12**
Fullscreen:
image

Resized:
image

On hovering over the home button:
image

Compatible with the menu slider:
image

### CODE REVIEW

**Comment by rdev12** [comment]
Note: I just reformatted the code in team.php (didn't do any edits)

**Comment by Darshilp326** [comment]
@RishikeshDevanathan Please solve conflicts!

**Comment by rdev12** [comment]
> @RishikeshDevanathan Please solve conflicts!

I've done it. Hope it doesn't break anything

File: .gitignore [hunk 1]
```diff
@@ -1 +0,0 @@
-/config.php
```
Comment (debarghya472) [inline .gitignore]: do not delete config.php from gitignore

File: team.php [hunk 2]
```diff
@@ -21,12 +21,7 @@
     <link rel="stylesheet" type="text/css" href="css/countdown.css?version=51" />
```
Comment (debarghya472) [inline team.php]: Create an error page independently. Do not make changes in team.php 
If u haven't pulled properly do so.

**Review by debarghya472 — CHANGES_REQUESTED** [review]
make the changes

File: team.php (hunk 2 above)
Comment (rdev12) [reply team.php]: I reformatted the code with shift+alt+f... I don't think I changed any code.

File: .gitignore (hunk 1 above)
Comment (rdev12) [reply .gitignore]: Should I commit gitignore files during pull requests? I am kinda new to this, sorry

**Review by rdev12 — COMMENTED** [review]
I clicked on Resolve Conversation for all. Is this what I'm supposed to do to resolve the problems? I apologize, this is my first time

**Review by debarghya472 — APPROVED** [review]
Nice job

**Pull request merged**